R/table_functions.R

```diff
@@ -52,13 +52,13 @@ add_orgnr <- function(data_list, by = "RHF") {
         data_list[["hospital_name_structure"]] %>%
           dplyr::select(
             .data[[org_column]],
-            .data[[config[["data"]][["column"]][["unit_id"]][["sh"]]]],
-            .data[[config[["data"]][["column"]][["unit_name"]][["sh"]]]]) %>%
+            .data[[config$data$column$unit_id$sh]],
+            .data[[config$data$column$unit_name$sh]]) %>%
           dplyr::mutate(
-            "OrgNrShus" = as.character(.data[[config[["data"]][["column"]][["unit_id"]][["sh"]]]])),
-        by = config[["data"]][["column"]][["unit_id"]][["sh"]]
+            "OrgNrShus" = as.character(.data[[config$data$column$unit_id$sh]])),
+        by = config$data$column$unit_id$sh
       ) %>%
-      dplyr:: filter(!is.na(.data[[config[["data"]][["column"]][["unit_name"]][["sh"]]]]))
+      dplyr:: filter(!is.na(.data[[config$data$column$unit_name$sh]]))
```

Comment:
Are these changes related to current PR? Maybe we should make a `config` PR where we fix all the `config` stuff?

Reply:
BTW, not a big deal... I am fine with merging the PR including these changes.

Reply:
I thought that since there is nothing major there, I would add it here. but I agree that it could have been gathered in a separate PR.